translation/locale_en.php

```diff
@@ -74,7 +74,8 @@
 //$template['index']['frforum']="french forum";
 //$template['index']['osxsite']="OS X builds";
 // other
-$template['index']['quickdnl']="quick download";
+//$template['index']['quickdnl']="quick download";
+$template['index']['quickdnl']="Grab latest build";
```

Comment:
I think the name `quickdnl` should be changed, because old string is incompatible with new one (by meaning). In translations, old text will be used, but new link, which is confusing.